frontend/src/routes/+layout.svelte

```diff
@@ -15,6 +16,7 @@
     if (browser) {
         try {
             authenticate();
+            getStocksDetails();
```

Comment:
Put getStocksDetails outside of this try / catch statement - at the moment it won't be called if the user fails to authenticate. We expect it to succeed, so it's okay to fail noisily if it doesn't.